src/cli/node/json_rpc_client.rs

```diff
@@ -99,9 +99,26 @@ pub fn get_balance(addr: SocketAddr, pkh: Option<PublicKeyHash>) -> Result<(), f
     );
     let response = send_request(&mut stream, &request)?;
     log::info!("{}", response);
-    let amount = parse_response::<u64>(&response)?;
-
-    println!("{} wits", Wit::from_nanowits(amount));
+    let amount = parse_response::<(u64, u64)>(&response)?;
+    if amount.0 >= amount.1 {
```

Comment:
This if is needed because `Wit` must be positive, and here we also want to print negative balance. Should we create a new `SignedWit` type that accepts negative amounts?